apps/question-the-docs/backend/documents/models.py

```diff
@@ -1,9 +1,18 @@
+import typing as t
+from enum import Enum
+
 from pydantic import BaseModel, Field
 
 
+class Repo(str, Enum):
```

Comment:
I don't think this should be an enum, because the list of sites to document is something that could and should be set in data, not code.